src/config.js

```diff
@@ -1,5 +1,8 @@
+const { SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN, SNOWPACK_PUBLIC_DATA_URL, SNOWPACK_PUBLIC_TRANSLATION_URL} = import.meta.env
+
+
 export default {
-  dataUrl: 'https://spreadsheets.google.com/feeds/list/1CyPozeKhmOuIaVnKDQAUKKsvA9R4F20hEQ3MSGWczP8/1/public/full?alt=json',
-  accessToken: 'pk.eyJ1IjoiamFjb2JyZGFsdG9uIiwiYSI6ImNrYXc2anFjbjIxNGwyeG14dWk0MDVycmIifQ.yILWplracVMB4mv1dWwTtg',
-  translationUrl: 'https://spreadsheets.google.com/feeds/list/1m5QPNP6O8nsQsqJcQbwib_obpXRjmbYYGtwKzzg1l38/1/public/full?alt=json'
+  dataUrl: SNOWPACK_PUBLIC_DATA_URL,
```

Comment:
We should try to add some kind of warning if these don't exist. Might want something like:
```diff
diff --git a/src/config.js b/src/config.js
index 69b0bda..62de88c 100644
--- a/src/config.js
+++ b/src/config.js
@@ -1,8 +1,14 @@
 const { SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN, SNOWPACK_PUBLIC_DATA_URL, SNOWPACK_PUBLIC_TRANSLATION_URL} = import.meta.env
 
+function ensure(variable_name, variable) {
+  if(import.meta.env.MODE === 'development') {
+    console.error(`${variable_name} must be defined in your environment. See https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations#environment-labels for details`)
+  }
+  return variable
+}
 
 export default {
-  dataUrl: SNOWPACK_PUBLIC_DATA_URL,
-  accessToken: SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN,
-  translationUrl: SNOWPACK_PUBLIC_TRANSLATION_URL
+  dataUrl: ensure("SNOWPACK_PUBLIC_DATA_URL", SNOWPACK_PUBLIC_DATA_URL),
+  accessToken: ensure("SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN", SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN),
+  translationUrl: ensure("SNOWPACK_PUBLIC_TRANSLATION_URL", SNOWPACK_PUBLIC_TRANSLATION_URL)
 }
```

Reply:
nice suggestion!